easypost/carrier_account.py

```diff
@@ -24,11 +38,16 @@ def types(cls, api_key: Optional[str] = None) -> List[str]:
         response, api_key = requestor.request(method=RequestMethod.GET, url="/carrier_types")
         return convert_to_easypost_object(response=response, api_key=api_key)
 
+    # Overrides CreateResource.create
     @classmethod
-    def register(cls, api_key: Optional[str] = None, **params) -> "CarrierAccount":
+    def create(cls, api_key: Optional[str] = None, **params) -> "CarrierAccount":
         """Creates a Carrier Account that requires custom registration (eg: FedEx & UPS)."""
         requestor = Requestor(local_api_key=api_key)
-        url = f"{cls.class_url()}/register"
+
+        if params.get("type") is None:
```

Comment:
mAkE iT pYtHoNiC 😉 

`params.get("type")` is defined twice in this function, let's assign it to a variable and reuse that.

Reply:
Resolved